src/components/magicui/MatterStacking.tsx

```
@@ -0,0 +1,584 @@
+          tokensContainer.style.top = '0';
+          tokensContainer.style.left = '0';
+          tokensContainer.style.width = '100%';
+          tokensContainer.style.height = '100%';
+          tokensContainer.style.overflow = 'hidden';
+          
+          // Use event delegation for click events
+          if (onTokenClick) {
+            tokensContainer.addEventListener('click', (e) => {
+              const target = e.target as HTMLElement;
+              const tokenElement = target.closest('.token-element') as HTMLElement;
+              if (tokenElement && tokenElement.dataset.token) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)